website/docs/react-native/santoku/decisions/adr-009-push-notification-fcm.mdx

```diff
@@ -245,8 +245,7 @@ FCMから以下のエラー応答が返ってきた場合には、通知送信
 - レスポンスヘッダにRetry-Afterが含まれている場合は、そのヘッダで指定された秒数だけ待機
 - レスポンスヘッダにRetry-Afterが含まれていない場合は指数バックオフに従い、1秒、2秒、4秒と指数関数的に時間を増やしながら待機
 
-このアプリでは、クライアント側となるモバイルアプリのサンプルを充実させることに注力しているため、
-今回はバックエンド側での再送制御までは実装しないものとします。
+このアプリでは、クライアント側となるモバイルアプリのサンプルを充実させることに注力しているため、[Firebase Admin SDKが自動的にリトライする仕様](https://firebase.google.com/docs/reference/admin/error-handling#automatic-retries)に準拠します。
```

Comment:
It is sometimes difficult to read the specifications from the documentation, so I would like comments that it is better to check with the source code.
For example, in Node.js, the `Retry-After` header does not seem to be taken into account, but looking at the actual source code, it seems that the retry time is [determined by the `Retry-After` value](https://github.com/firebase/firebase-admin-node/blob/72928b5fea3cad303361494a67de2ea786bb4171/src/utils/api-request.ts#L337).

Reply:
I've added it, please check it out.
5ab8b9c9733c2a7a11cb35c453a977e2bb9225ba